src/libm/CMakeLists.txt

```diff
@@ -84,13 +97,14 @@ add_custom_command(
 add_custom_target(headers ALL DEPENDS ${SLEEF_INCLUDE_HEADER} ${RENAME_HEADERS})
 
 # Build main library
-add_library(${TARGET_LIBSLEEF}
-  SHARED sleefdp.c sleefsp.c
-  $<TARGET_OBJECTS:${OBJECT_SSE2}>
-  $<TARGET_OBJECTS:${OBJECT_SSE4}>
-  $<TARGET_OBJECTS:${OBJECT_AVX}>)
+add_library(${TARGET_LIBSLEEF} SHARED sleefdp.c sleefsp.c)
 add_dependencies(${TARGET_LIBSLEEF} headers)
 
+target_sources(${TARGET_LIBSLEEF} PRIVATE $<$<BOOL:${COMPILER_SUPPORTS_SSE2}>:$<TARGET_OBJECTS:${OBJECT_SSE2}>>)
```

Comment:
Is the `$<$<BOOL:${COMPILER_SUPPORTS_SSE2}>:$<TARGET_OBJECTS:${OBJECT_SSE2}>>` conditional inclusion? If so, may I ask you to add a comment before this line about that?